index.html

```diff
@@ -40,6 +45,164 @@ <h2 class="headline-subHeading">I’m a Software Developer</h2>
         </article>
     </section>
 
+    <section id="Works">
+        <header class="works-Title">My Recent Works<hr></header>
+
+        <article class="card project1">
+            <header class="project-Header">
+                <img src="images/p1-img.png" alt="project-1 image">
+                <h1>Multi-Post Stories</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                    standard dummy text.</p>
+            </header>
```

Comment:
- [ ] There should only be one `<header>` section on an HTML page for semantic reasons. I therefore suggest that you replace all `<header>` tags with `<div>` or `<section>` tags

> The `<header>` element is used to identify content that precedes the primary content of the web page and often contains website branding, navigation elements, search forms, and similar content that is duplicated across all or most pages of a website